promgen/templates/promgen/service_action_button_group.html

```diff
@@ -0,0 +1,49 @@
+{% load i18n %}
+{% load promgen %}
+
+
+<div style="display: inline-block;">
+  <div class="btn-group btn-group-sm" role="group" aria-label="...">
+    <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+      {% translate "Actions" %} <span class="caret"></span>
+    </button>
+    <ul class="dropdown-menu">
+
+      <li>
+        <a href="">
+          <form action="{% url 'service-notifier' service.id %}" style="display:inline" method="post" v-pre>{% csrf_token %}
+            <input type="hidden" name="sender" value="promgen.notification.user">
+            <input type="hidden" name="value" value="{{request.user.username}}" />
+            <button style="background:none;border:none;padding:0;">{% translate "Subscribe to Notifications" %}</button>
+          </form>
+        </a>
+      </li>
+
+      <hr>
+
+      <li role="presentation"><a href="{% urlqs 'audit-list' service=service.id %}">{% translate "Edit History" %}</a></li>
+      <li role="presentation"><a href="{% urlqs 'alert-list' service=service.name %}">{% translate "Alert History" %}</a></li>
+
+      <hr>
+
+      <li role="presentation"><a href="{% url 'api:service-rules' name=service.name %}">{% translate "Export Rules" %}</a></li>
+      <li role="presentation"><a href="{% url 'api:service-targets' name=service.name %}">{% translate "Export Service" %}</a></li>
+
+      <hr>
+
+      <li role="presentation">
+        <a href="">
+          <form method="post" action="{% url 'service-delete' service.id %}" onsubmit="return confirm('{% translate "Delete this service?" %}')" style="display: inline">
+            {% csrf_token %}   
+              <button style="background:none;border:none;padding:0;color:#d9534f" >{% translate "Delete Service" %}</button>  
+          </form>
+        </a>
+      </li>
+
+    </ul>
+  </div>
```

Comment:
Even though `<hr>` is more semantic, I think using bootstrap's `<li role="separator" class="divider"></li>` will result in a cleaner layout

Before
<img width="255" alt="before" src="https://github.com/user-attachments/assets/d16f8f2a-8881-4671-9148-f0db985d6cca">

After
<img width="257" alt="after" src="https://github.com/user-attachments/assets/a376f754-7d01-4889-90b2-bc164c2effcd">

Reply:
<img width="776" alt="image" src="https://github.com/user-attachments/assets/eda8b4e4-bbc9-4e34-a7f6-cc25c3380a8d">
As you can see in the image attached to the PR description, I have already set the hr tag style, and the original divider style should look like this. . But somehow it didn't work at your local..
I update the hr style again